README.md

```diff
@@ -158,6 +158,15 @@ class _NavigationSampleState extends State<NavigationSample> {
 }
 ```
 
+#### Task Removed Behavior
+
+The taskRemovedBehavior parameter of navigation session initialization defines how the navigation should behave when a task is removed from the recent apps list on Android. It can either:
+
+    CONTINUE_SERVICE: Continue running in the background. (default)
```

Comment:
These enum names to don match the names in enum:
TaskRemovedBehavior.continueService
TaskRemovedBehavior.quitService